storage/postgres/repository.go

```diff
@@ -76,18 +76,22 @@ func (r *Store) UpdateSnapshotLatestVersion(ctx context.Context, snapshot *model
 }
 
 // GetSnapshotByFields returns full snapshot data
-func (r *Store) GetSnapshotByFields(ctx context.Context, namespace, name, version string, latest bool) (*models.Snapshot, error) {
+func (r *Store) GetSnapshotByFields(ctx context.Context, namespace, name, version string, latest *bool) (*models.Snapshot, error) {
 	sh := &models.Snapshot{
 		Namespace: namespace,
 		Name:      name,
 	}
 	var query strings.Builder
 	var args []interface{}
-	query.WriteString(`SELECT id, version, latest from snapshots where namespace=$1 and name=$2 and latest=$3`)
-	args = append(args, namespace, name, latest)
+	query.WriteString(`SELECT id, version, latest from snapshots where namespace=$1 and name=$2`)
```

Comment:
SQLs like this are prone to SQL injection.
Should we parameterize the SQL?
 If not, validate the params?

Reply:
@chakravarthyvp we are parameterizing the SQL I think? Why do you think we are not doing it?

Reply:
Usually the SQL driver provides the functions with query parameters (internally validated). If the pg go module does not provide these, then we need to validate it explicitly & raise issue upto the API.
for ex., 
`SELECT id, version, latest from snapshots where namespace=$1 and name=$2` where `name` is technically coming through the API request.

Reply:
Here the postgres driver that we're using does execute this in prepared statement only. AFAIK prepared statement does avoid SQL injection.